src/com/tutorial/main/Handler.java

```diff
@@ -1,51 +0,0 @@
-package com.tutorial.main;
-
-import java.awt.Graphics;
-import java.util.ArrayList;
-import java.util.LinkedList;
-
-public class Handler {
-	
-	LinkedList<GameObject> object = new LinkedList<GameObject>();
```

Comment:
Define this as `List<GameObject> object = new ArrayList<GameObject>();`
Two things here:
1 - Your defined object should be as generic as possible for polymorphism's sake. It will allow you to change the type of list you use within the code and not break anything else that's being done.
2 - ArrayLists are generally preferred since they have faster random reads and is faster to add objects to the end of the list. You'll lose out on removals though, so in this class it's almost a draw in terms of efficiency.